tasks/RedHat/generic/configure-tuned.yml

```diff
@@ -35,5 +35,52 @@
 
   when:
     - sap_hana_preconfigure_switch_to_tuned_profile_sap_hana
+    - (ansible_architecture != "ppc64le" or (ansible_architecture == "ppc64le" and ansible_distribution_version != "8.0"))
+
+- block:
+    - name: On RHEL 8.0 on ppc64le, create directory /etc/tuned/sap-hana-ppc64le
+      file:
+        path: /etc/tuned/sap-hana-ppc64le
+        state: directory
+        owner: root
+        group: root
+        mode: '0755'
+
+    - name: On RHEL 8.0 on ppc64le, copy file /etc/tuned/sap-hana-ppc64le/tuned.conf
+      copy:
+        src: etc/tuned/sap-hana-ppc64le/tuned.conf
+        dest: /etc/tuned/sap-hana-ppc64le/tuned.conf
+        owner: root
+        group: root
+        mode: '0644'
+        backup: yes
+
+    - name: On RHEL 8.0 on ppc64le, show active tuned profile again
+      command: bash -lc "/usr/sbin/tuned-adm active | grep ":" | cut -d ":" -f 2 | awk '{$1=$1;print}'"
+      check_mode: no
+      register: current_profile_ppc64le
+      changed_when: false
+    - debug:
+        var: current_profile_ppc64le.stdout_lines, current_profile_ppc64le.stderr_lines
+
+    - name: On RHEL 8.0 on ppc64le, switch to tuned profiles sap-hana sap-hana-ppc64le if not currently active
+      block:
+        - name: Switch to tuned profiles sap-hana sap-hana-ppc64le if not currently active
+          shell: /usr/sbin/tuned-adm profile sap-hana sap-hana-ppc64le
```

Comment:
you can use `command` here instead of `shell`

Reply:
+1 :-)

Reply:
done - https://github.com/berndfinger/sap-hana-preconfigure/commit/318a2c6ec8d7a63efd457e7f2b39bd33326d41ab